FiveCalls/FiveCalls/IssuesContainerViewController.swift

```diff
@@ -38,15 +39,40 @@ class IssuesContainerViewController : UIViewController, EditLocationViewControll
         
         issuesVC.didMove(toParentViewController: self)
         issuesViewController = issuesVC
+        
+        let effectView = UIVisualEffectView(frame: headerContainer.bounds)
+        effectView.translatesAutoresizingMaskIntoConstraints = false
+        effectView.effect = UIBlurEffect(style: .light)
```

Comment:
I like to extract these into lazy properties, like this:

```swift
lazy var effectView: UIVisualEffectView = {
    let effectView = ...
    // more setup
    return effectView
}
```

Then you can just use it in `viewDidLoad` or wherever and they get set up properly.